src/db/modelsType.ts

```diff
@@ -0,0 +1,22 @@
+export interface OrderAttributes {
```

Comment:
As I see, these are your models. So you can just call them `Order`, `Product`, etc...